backend/tests/routes/test_documents.py

```diff
@@ -199,6 +202,7 @@ def create_4_documents(test_db, client, superuser_token_headers):
     document4_payload = {
         "publication_ts": "1997-01-01T00:00:00.000000+00:00",
         "name": "Energy Sector Strategy 1387-1391 (2010/8-2015/3)",
+        "postfix": "postfix4",
```

Comment:
We do probably also need to test out the option of `postfix=None`

Reply:
good point